bokehjs/src/lib/models/glyphs/annular_wedge.ts

```diff
@@ -134,18 +134,12 @@ export class AnnularWedgeView extends XYGlyphView {
     generic_area_legend(this.visuals, ctx, bbox, index)
   }
 
-  private _scenterxy(i: number): {x: number, y: number} {
+  scenterxy(i: number): [number, number] {
     const r = (this.sinner_radius[i] + this.souter_radius[i])/2
     const a = (this._start_angle[i]  + this._end_angle[i])   /2
-    return {x: this.sx[i] + (r*Math.cos(a)), y: this.sy[i] + (r*Math.sin(a))}
-  }
-
-  scenterx(i: number): number {
-    return this._scenterxy(i).x
-  }
-
-  scentery(i: number): number {
-    return this._scenterxy(i).y
```

Comment:
We should do some investigation of our own to make sure Dask, etc are not relying on these. Or leave them in with a deprecation and a note in the migration guide.